vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -19,13 +19,25 @@ namespace Microsoft.ReactNative.Bridge
     String Name { get; };
 
     Windows.UI.Xaml.FrameworkElement CreateView();
+  };
 
+  [webhosthidden]
+  interface IViewManagerWithExportedViewConstants
+  {
     Windows.Foundation.Collections.IMapView<String, IInspectable> ExportedViewConstants { get; };
+  }
 
+  [webhosthidden]
+  interface IViewManagerWithNativeProperties
+  {
     Windows.Foundation.Collections.IMapView<String, ViewManagerPropertyType> NativeProps { get; };
 
     void UpdateProperties(Windows.UI.Xaml.FrameworkElement view, Windows.Foundation.Collections.IMapView<String, IInspectable> propertyMap);
```

Comment:
Should it be NativeProps and UpdateProps? Or NativeProperties and UpdateProperties? Them being mismatched seems strange.